src/components/Icon/Icon.tsx

```diff
@@ -172,9 +172,11 @@ export type IconSource =
   | SVGSource
   | 'placeholder'
   | BundledIcon
-  | UntrustedSVG;
+  | UntrustedSVG
+  | React.SFC<React.SVGProps<SVGSVGElement>>;
 export interface Props {
   /** The SVG contents to display in the icon. Icons should be in a 20 X 20 pixel viewbox */
+  /** @deprecated Passing a React Element as source is deprecated. Pass a React Component instead */
```

Comment:
It's my understanding that this prop itself is not actually deprecated, only its ability to receive a React Element. Can you clarify? If that's the case, I don't think we want the `deprecated` tag here, only the warning on line 239.

Reply:
You're right, the prop itself isn't deprecated, just part of its interface 👍

Reply:
In that case the `deprecated` tag should be removed here